IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -227,9 +228,10 @@ - (void)executeRequestImpl:(MSIDRequestCompletionBlock)completionBlock
         
         [self tryRefreshToken:refreshToken tokenType:tokenType completionBlock:completionBlock];
     }];
+    
 }
 
-- (void)fetchCachedTokenAndCheckForFRTFirst:(BOOL)checkForFRT shouldComplete:(BOOL)shouldComplete completionHandler:(void (^)(MSIDBaseToken<MSIDRefreshableToken> *, MSIDRefreshTokenTypes, NSError *))completionHandler
+- (void)fetchCachedRefreshTokenWithCompletionHandler:(void (^)(MSIDBaseToken<MSIDRefreshableToken> *, MSIDRefreshTokenTypes, NSError *))completionHandler
```

Comment:
I think these changes are not part of throttling. Possible to pull latest dev?

Reply:
I already pull latest dev, however there is an error when merging code, some one overwrited the logic from dev. I will recover it back.

Reply:
Addressed